README.md

```diff
@@ -53,7 +53,7 @@ bin/dev
 rake demo_data:reset
 ```
 
-And visit http://localhost:3000 to see the app. You can use the following credentials to log in (generated by DB seed):
+And visit http://localhost:3000 to see the app. You can also set a custom port with the `PORT` variable in .env file. You can use the following credentials to log in (generated by DB seed):
```

Comment:
I think we can remove this and leverage the `.env.example` file as documentation for new devs